src/styles/body.scss

```diff
@@ -0,0 +1,9 @@
+
+body {
+  margin: 0;
+  height: 100vh;
+  flex-direction: column;
```

Comment:
The `flex-direction` property is used with `display: flex` or `display: inline-flex`. Since `body` doesn't seem to be a flex container for other elements (no indication of it having flex items), this property might not be needed. If `body` is indeed a flex container, please disregard this comment. Otherwise, consider removing the `flex-direction` property if it's not being used.